src/_layouts/blog.njk

```diff
@@ -6,11 +6,14 @@ layout: page-default
 <article class="blog-post content__inner--full-width">
     <!-- header title + image .etc -->
     <header class="blog-post__header">
-        <div class="blog-post__header__image">

```

Comment:
I think, however we do it, the hero image should always be required.

Reply:
If you want this to always be required then we need to spin up another layout for the blog or other static pages where a hero image is not required. In this instance I think having one of the CoC is a bit weird which is why I made it variable but we can spin out other layout as an enhancement if it is more explicit.